core/camel-core-model/src/main/java/org/apache/camel/model/ResumableDefinition.java

```diff
@@ -40,6 +41,13 @@
     @Metadata(required = true, javaType = "org.apache.camel.ResumeStrategy")
     private String resumeStrategy;
 
+    @XmlTransient
+    private LoggingLevel loggingLevel = LoggingLevel.WARN;
```

Comment:
See hos we do this in LogDefinition with just a String type (not a Ref name)

org.apache.camel.model.LogDefinition

Reply:
I had a mistake too ;)
https://github.com/apache/camel/commit/55bc0f2430fa2df1569a8c70b853e0d9e2c5acd3

Reply:
:D Nice, thanks for the tip and I glad you found an issue!